dropwizard-configuration/src/test/resources/factory-test-invalid.json

```diff
@@ -0,0 +1,3 @@
+{
+	"name": "Boop"
+}
```

Comment:
I think I did that now but I am not sure if my GIT client removed them again. The github view is confusing for these cases.